content/vault/v1.20.x/content/api-docs/system/internal-counters.mdx

```diff
@@ -211,6 +211,17 @@ The `start_time` in the response is the earliest time there is activity data
 in the queried period. The `end_time` is the end of the final month of the queried
 period.
 
+<Note>
+
+This API supports only monthly granularity until support for daily-level is added. Until then, 
+the `start_time` and `end_time` values in the reponse are normalized to align with full calendar months. 
+This normalization occurs whether the times are explicitly provided in the request or defaulted by Vault.
+
+- The `start_time` is normalized to the start of the month in which the effective `start_time` falls.
+- The `end_time` is normalized to the end of the month in which the effective `end_time` falls.
+
+</Note>
```

Comment:
```suggestion
The `/sys/internal/counters` endpoint only supports monthly granularity. As a
result, the `start_time` and `end_time` values in API reponses are normalized to
align with full calendar months.

Normalization occurs whether you provide explicitly times in the request or use
the defaults:

- `start_time` normalizes to the start of the month in which the effective `start_time` falls.
- `end_time` normalizes to the end of the month in which the effective `end_time` falls.
```

Style correction: write in active voice, limit use of asides, focus on current functionality rather than future functionality